docs/intro.md

```diff
@@ -68,7 +68,7 @@ build a local project against the libraries in `snap/stage` by running `snapcraf
 shell make`. Though in general, you are encouraged to add even local
 projects to snapcraft.yaml with a local `source:` path.
 
-For rapid iteration one can run `snappy try` against this directory to have it
+For rapid iteration one can run `sudo snap try` against this directory to have it
```

Comment:
I think we are avoiding the `sudo` because snap could work with login. Maybe ask @caldav?